packages/react/src/hooks/utils/useController.ts

```diff
@@ -1,10 +1,13 @@
+import { CoreHelpers } from '@web3modal/core'
 import { useEffect, useMemo, useState } from 'react'
 import { useClientInitialized } from '../data/useClientInitialized'
 
 // -- types ----------------------------------------------------- //
+type NullishReturn<TReturn> = TReturn | null
```

Comment:
Name's kinda confusing, maybe we could stick to normal standards and call it Nullable?

Reply:
Totally, no idea why I made it so complex 😂 changed